src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -123,13 +124,34 @@ export class ReviewRoute extends Component<ReviewProps> {
             }
           })
       } else {
-        this.props.order.conversation && !this.props.isEigen
-          ? this.props.router.push(
-              `/user/conversations/${this.props.order.conversation.internalID}`
-            )
-          : this.props.router.push(
-              `/orders/${this.props.order.internalID}/status`
-            )
+        const { order, router, user, isEigen } = this.props
+
+        if (!userHasLabFeature(user, "Make Offer On All Eligible Artworks")) {
```

Comment:
we can improve the if's here once the feature flag is retired, the isEigen check can be done before all and make everything easier to read